configman/option.py

```diff
@@ -95,6 +96,7 @@ def __init__(
         self.not_for_definition = not_for_definition
         self.reference_value_from = reference_value_from
         self.secret = secret
+        self.change_flag = change_flag
```

Comment:
the ConfigurationManager object reaches in and changes the value, during the overlay process.  If it were marked as private, it would be a conspicuous violation of the Option's privacy.